### ISSUE TITLE
[Adaptive Cards-More Option]: At 200% resize settings, tick mark visual cue present for the 'Use fixed dimensions' menu item which appears on after activating the 'More (...)' icon control is not visible.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9215]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
1. Go to system Settings-> System-> Display-> Scale & Layout-> Change the size of text, apps, and other items at 100%-> Display Resolution (1680*1050)
2. Go to system Settings-> Accessibility-> Text size-> 200

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'More/Less (...)' icon button present on the header section of the application using tab key and activate it using enter key.
3. Now navigate to the 'Use fixed dimensions' menu item using tab key and activate it using enter key.
4. Verify that, at 200% resize settings, tick mark visual cue present for the 'Use fixed dimensions' menu item which appears on after activating the 'More (...)' icon control is not visible or not.

**Actual Result:**
At 200% resize settings, tick mark visual cue present for the 'Use fixed dimensions' menu item which appears on after activating the 'More (...)' icon control is not visible.

**Expected Result:**
At 200% resize settings, tick mark visual cue present for the 'Use fixed dimensions' menu item which appears on after activating the 'More (...)' icon control should be properly visible.

**User Impact:**
Low vision users depend on the resize settings will be impacted if at 200% resize settings, tick mark visual cue present for the 'Use fixed dimensions' menu item which appears on after activating the 'More (...)' icon control is not visible due to which users will be unable to know that the 'Use fixed dimensions' menu item is selected or not and users might try to activate the same control multiple times without seeing any visual cue. 

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/83afb5c3-f101-41bc-a5b7-66458e613f22)

### Card JSON

```JSON
N/A
```
